README.md

```diff
@@ -74,22 +73,22 @@ Once complete:
 ```
 cd src/
 ```
-and start Omni Core using ```./mastercored``` (or ```./qt/mastercore-qt``` if built with UI). The inital parse step for a first time run
+And start Omni Core using `./bitcoind` (or `./qt/bitcoin-qt` if built with UI). The inital parse step for a first time run
```

Comment:
Yeah of course. Wasn't sure, whether to reflect the current state, or where we're going. Given that it's pretty clear, I think I should edit it and use `omnicore`.